country_assignment.rb

```diff
@@ -0,0 +1,73 @@
+
+module CountryModule
+    def check_loan()
+       if(@population > 100000 && @gdp < 5)
+        puts "Country #{@name} will get a loan."
+       else
+        puts "Country #{@name} is not eligible for loan."
+       end
+    end
+
+    def chance_in_UNS_council()
+        if(@is_developed)
+            puts "Country #{@name} will get a chance in UNS council."
+        else
+            puts "Country #{@name} will not get a chance in UNS council."
+        end
+        
+    end
+
+    def win_war()
```

Comment:
Same here, method names should be more meaningful.

Reply:
> Same here, method names should be more meaningful.

/home/josh/truptij_training/ror_training/country_assignment.rb

Resolved now